### ISSUE TITLE
[MAINT] sol.vector.mask.mask_to_poly_geojson() doesn't save anything to output_path

### ISSUE THREAD

**Issue by guischmitd** [CosmiQ/solaris#209]
**TL;DR** `mask_to_poly_geojson()` function receives an argument called `output_path` (defaults to `None`) but there is no code for actually writing the output to the specified path.

First and foremost, thank you for this repo, there are so many useful functions here I can't even show all my gratitude! 
Also it's good to note that I've just started exploring solaris and still haven't gone through much of the code. This is also my first issue on a github repo, please correct me if I've made any mistakes, *feedback is awesome!*.
Even though I'm kind of new to open-source, I've been working with rasterio and fiona before finding solaris and believe I can contribute if needed.

## Maintenance request summary
Not sure if this is intended behaviour. The `mask_to_poly_geojson()` function receives an argument called `output_path` (defaults to `None`) but there is no code for actually saving the output to the specified format ( `'csv'` or `'geojson'`), *even though* it also takes a `reference_im` as input. 
On my application I had to take the returned geodataframe, convert each polygon point to EPSG coordinates based on my raster tiles and save a .shp with fiona, *even though* this [tutorial](https://solaris.readthedocs.io/en/latest/tutorials/notebooks/api_mask_to_vector.html) and the function's docstring state that it saves the output if a path is specified.

## Task detail and notes
- [ ] Determine whether this is intended behaviour **OR** saving should be part of the function
- [ ] Fix tutorials and documentation in the first case, implement saving if the latter is true

#### Any additional information:
I'm using solaris version 0.1.2 installed via pip since I don't use conda and already had GDAL 2.4.0 installed for other projects.

- What other code will this impact?
I'm not sure, maybe someone with more knowledge of the repo can help me here?
- Do new tests need to be written or do existing tests need to be updated? 
Same as above
- Will this impact documentation?
Depends on the intended behaviour, but the documentation is wrong as it stands now.

**Comment by nrweir** [comment]
Hi @guischmitd,

Thanks for catching this - we’ll get it fixed for the next release.

**Comment by nrweir** [comment]
@guischmitd Later than anticipated, but this is resolved in #244.
